charts/rbac/templates/client/cluster.yaml

```diff
@@ -32,4 +32,18 @@ subjects:
   kind: Group
   name: {{ .Values.rbac.client.name }}:view
 
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: keycloak:developer
```

Comment:
it's better to use oidc, we may use okta or authentik in future and this role binding might confuse